docs/reference/technology/connectivity/connectivity.md

```diff
@@ -48,7 +48,7 @@ Cellular coverage, load balancing and reliability are the responsibility of the
 
 The new cellular technologies, NB-IoT and CAT-M1, are designed for IoT devices. They offer some advantages, but there are also risks and costs.
 
-**Narrow-band-IoT** (NB-IoT) relies on a simple waveform and therefore requires the lowest power consumption of all the cellular methods. It also has fewer components and offers better signal penetration through walls (thanks to lower bitrates and better link budgets). NB-IoT requires country-specific chips because each country uses its own spectrum, which complicates production. It also causes problems with large transfers because of its reduced bandwidth.
+**Narrow-band-IoT** (NB-IoT) NB-IoT is a 3GPP standard for low-power wide area networks (LPWANs) specifically designed for low-cost, battery-powered IoT devices. The two network attach options for NB-IoT are network attach with and without packet data network (PDN). NB-IoT relies on a simple waveform and therefore requires the lowest power consumption of all the cellular methods. It also has fewer components and offers better signal penetration through walls (thanks to lower bitrates and better link budgets). NB-IoT requires country-specific chips because each country uses its own spectrum, which complicates production. It also causes problems with large transfers because of its reduced bandwidth.
```

Comment:
A sentence like "NB-IoT relies on a simple waveform and therefore requires the lowest power consumption of all the cellular methods" seems much higher level than "The two network attach options for NB-IoT are network attach with and without packet data network (PDN)".

Reply:
Would switching the order help, or is it better to rephrase entirely?

Reply:
Try it both ways and see :D